querydsl-libraries/querydsl-sql-spatial/pom.xml

```diff
@@ -93,6 +93,12 @@
       <version>2023.1.0</version>
       <optional>true</optional>
     </dependency>
+    <dependency>
+      <groupId>net.postgis</groupId>
+      <artifactId>postgis-jdbc-jts</artifactId>
+      <version>2023.1.0</version>
```

Comment:
Hi @vidakovic I'm not comfortable at this stage on dropping java 11 support.

Any change on using an older version of postgis that is 11 compatible?

Reply:
Have to see if that exists... I just know that the dependencies referenced originally are quite different (package names changed!). Let me see if I can find a version with the new package names (and features) that is JDK 11 compatible.